pages/iframe/custom.mdx

```diff
@@ -0,0 +1,32 @@
+import { Tab, Tabs } from "nextra-theme-docs";
+
+# Custom Implementation
+
+This method is recommended for users who are looking to do more dynamic things in their iframe.
+The current implementation of the URL relies on Alchemy's indexer to render the image. However,
+if you require assets from a custom api then this is the method for you. For example, the [Sapienz](https://opensea.io/assets/ethereum/0x26727ed4f5ba61d3772d1575bca011ae3aef5d36/1)
+project has a custom api that sorts the order of images based on what is equipped by the owner.
+
+## Steps
+
+1. Fork and clone the [iframe](https://github.com/tokenbound/iframe) repository:
+   `git clone git@github.com:githubUser/iframe.git`
+2. Run install with `pnpm`, `yarn`, or `npm`.
+3. Create a `.env.local` file and copy the variables from `.env.example`
+4. Create an account for [Alchemy](https://dashboard.alchemy.com/) and create an API key.
+5. Add in the the Alchemy key to the `NEXT_PUBLIC_ALCHEMY_KEY` and `NEXT_PUBLIC_PROVIDER_ENDPOINT` env variables. `NEXT_PUBLIC_PROVIDER_ENDPOINT` will be the `https` link.
+6. Create an account for [n.xyz](https://n.xyz/) and api key. This is for checking token approvals.
+7. Add in the n.xyz api key to `NEXT_PUBLIC_NXYZ_API_KEY`. Make sure you enable cors for `n.xyz` for the api key to your domain or `*` for all domains.
+8. Now add in your custom api endpoint to fetch your NFT's assets to `NEXT_PUBLIC_NFT_ENDPOINT`. The api should return an array of strings as image links. Or just a string as an image link. Without this env variable, the app will assume you want your images fetched from alchemy.
+9. To test this you can put in your the url `/contractAddress/tokenId` when running `pnpm dev` | `yarn dev` | `npm run dev`.
```

Comment:
@huynhr to be clear, you're saying that the endpoint can be tested at ex. 

```https://localhost:3000/0x..../15```

?

Reply:
(Or whatever port number the app runs on)

Reply:
Yeah I would flip this around to say something more explicit like
````
To test, start your local development server using pnpm dev... etc
then navigate to https://localhost:3000/0x..../[tokenID] to view the token
```